sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTestBase.java

```diff
@@ -1109,14 +1113,14 @@ void analyzeLinkedEntityRecognitionRunner(BiConsumer<List<String>, TextAnalytics
             LINKED_ENTITY_INPUTS,
             new TextAnalyticsActions()
                 .setDisplayName("Test1")
-                .setRecognizeLinkedEntitiesOptions(new RecognizeLinkedEntitiesOptions()));
+                .setRecognizeLinkedEntitiesActions(new RecognizeLinkedEntitiesAction()));
     }
 
     void analyzeSentimentRunner(BiConsumer<List<String>, TextAnalyticsActions> testRunner) {
         testRunner.accept(
             SENTIMENT_INPUTS,
             new TextAnalyticsActions()
-                .setAnalyzeSentimentOptions(new AnalyzeSentimentOptions()));
```

Comment:
Not sure I saw in the PR where the *Options types were removed from the code. Are they still used internally?

Reply:
The option bag `AnalyzeSentimentOptions` is still exist in Java side now. I haven't remove it from the package. The ***Action new type introduced should be used in the TextAnalyticsActions

Reply:
Ohh got it - they still need to exist for the non-action endpoints. When I read the changelog I thought they were getting removed.

Reply:
What about entities, linked entities, and key phrases? In .NET I completely removed those 3 types and the respective overloads in the client as currently there is no need to have the types.
Are you thinking on leaving them in Java?

Reply:
I will remove it in the next PR.

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/22335